dpsim-models/include/dpsim-models/MNASimPowerComp.h

```diff
@@ -4,6 +4,7 @@
 
 #include <dpsim-models/SimPowerComp.h>
 #include <dpsim-models/Solver/MNAInterface.h>
+#include <dpsim-models/MNAStampUtils.h>
```

Comment:
Did you use `git-clang-format`?

Reply:
Ooops, I did not apply it to this file😅. How did you figure out? By the non-alphabetic order? 😀

Reply:
Yes :grin: but is not a problem, is just trying to take advantage that we touch the file so we make it pretty